docs/userguide/index.rst

```diff
@@ -23,6 +23,7 @@
     optimizing
     debugging
     concurrency/index
+    workerpool
```

Comment:
On second thought, this isn’t the best place.
I think inside the https://docs.celeryq.dev/en/latest/userguide/workers.html would make more sense.